embed-v2.js

```diff
@@ -95,6 +95,7 @@
 
   .emgithub-file-dark {
     border: 1px solid #555;
+    background-color: #0d1117;
```

Comment:
Don't use the `black` color. Instead use the dark color from https://github.com/sindresorhus/github-markdown-css/blob/main/github-markdown-dark.css#L7.

In this way, the whole background is shown in `#0d1117` color.